src/gui/plugins/scene3d/Scene3D.hh

```diff
@@ -458,6 +471,12 @@ inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
     signals: void FollowTargetChanged(const std::string &_target,
         bool _waitForTarget);
 
+    /// \brief When fire, the ModelSDF associated to en entity should be
+    /// updated.
+    /// \param[in] _entity Entity to be update.
+    /// \param[in] _scale The scaling vector.
```

Comment:
```suggestion
    /// \brief When fired, the ModelSDF associated to an entity should be
    /// updated.
    /// \param[in] _entity Entity to be updated.
    /// \param[in] _scale The scaling vector.
```

Reply:
Updated in 2558167.

Reply:
> Updated in 2558167.

I think this fix may have been accidentally skipped. The commit resolves https://github.com/ignitionrobotics/ign-gazebo/pull/882#discussion_r668284600 but not the typos

Reply:
You're right. Fixed in f46616a.